query-engine/core/src/executor/mod.rs

```diff
@@ -90,3 +89,33 @@ pub trait TransactionManager {
 pub fn get_current_dispatcher() -> Dispatch {
     tracing::dispatcher::get_default(|current| current.clone())
 }
+
+tokio::task_local! {
+    static REQUEST_NOW: prisma_value::PrismaValue;
+}
+
+/// A timestamp that should be the `NOW()` value for the whole duration of a request. So all
+/// `@default(now())` and `@updatedAt` should use it.
+///
+/// That panics if REQUEST_NOW has not been set with with_request_now().
```

Comment:
Oh, this is cool actually. Better to panic than write a wrong date in a new operation!

Reply:
The only problem I see is somebody adds a new operation, does not test it with `now()` or `updatedAt` and the panic is triggered by a user.